report_pre_thesis/src/sections/related_work.tex

```diff
@@ -1,14 +1,43 @@
+The research suggests a balance between dataset size and practicality, indicating that while more data improves accuracy, the returns diminish against the costs of data acquisition\cite{dyar_effect_2021}.
+
+The research on the effect of training set size offers relevant considerations for our work. Their analysis of different error metrics for varying dataset sizes provides a good framework when analyzing the balance between data quantity and model structure in our efforts to minimize overall error
+
+\citeauthor{castorena_deep_2021} developed a deep spectral convolutional neural network (CNN) for processing and analyzing laser-induced breakdown spectroscopy (LIBS) signals.
+Their CNN is designed to first remove uncertainties from sensor data and then assess the chemical content qualitatively and quantitatively from a sample's LIBS signal.
+Their experiments showed that their method surpasses existing ones used by the Mars Science Laboratory for pre-processing and calibration.
+Notably, their CNN operates in real-time and does not require additional information like dark current, system response, temperature, or range.
+For chemical content estimation, their CNN matches current techniques' performance and even exceeds them when utilizing an end-to-end learning approach, all the while reducing prior information requirements and being ready for immediate deployment\cite{castorena_deep_2021}.
+
+Their development of a deep spectral CNN for analyzing LIBS signals showcases an innovative method that excels in real-time operation and minimal data dependency. Their method's ability to operate in real-time and with reduced data dependency provides a compelling framework for future considerations in our work, especially when evaluating the balance between model complexity and operational efficiency in reducing overall error.
+
+\citeauthor{chen_xgboost_2016} introduced XGBoost, a scalable end-to-end tree boosting system that has gained widespread use among data scientists for its state-of-the-art performance in various machine learning challenges. The system is particularly notable for its novel sparsity-aware algorithm tailored for sparse data and the introduction of a weighted quantile sketch method for approximate tree learning. These advancements allow XGBoost to efficiently handle large-scale data with significantly reduced resource requirements. Utilizing cache access patterns, data compression, and sharding, the authors were able to significantly enhance the scalability and efficiency of XGBoost. These approaches enabled it to process a large number of examples with fewer resources compared to other exisiting systems\cite{chen_xgboost_2016}.
```

Comment:
There's a rather high similarity to this and the XGBoost abstract. Don't want to get caught plagiarizing :)

Not saying that's what happened, but rather that a system may think so, due to the amount of words that overlap in the same order.